### PR TITLE
TheTheme Footer

### PR DESCRIPTION
**Pull request by agriffard**
Add css to TheTheme to display correctly the Header and Footer with a fixed navbar.

### CODE REVIEW

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <link type="image/x-icon" rel="shortcut icon" href="/TheTheme/favicon.ico" />
     <script asp-name="bootstrap" use-cdn="true" version="4" at="Foot"></script>
     <style asp-name="bootstrap" use-cdn="true" version="4"></style>
+    <style asp-src="/TheTheme/css/sticky-footer-navbar.css"></style>
```
Comment (sebastienros) [inline src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml]: Why do we need that? Isn't Bootstrap 4 able to render a simple theme by itself?

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml [hunk 2]
```diff
@@ -36,6 +37,7 @@
             @await RenderBodyAsync()
         </div>
     </main>
+    @await RenderSectionAsync("Footer", required: false)
```
Comment (sebastienros) [inline src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml]: What do we have in the footer? Or is it just for consistency with the other themes?

Comment (agriffard) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml]: Yes, I'd like all the themes have the same zones.

File: src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml (hunk 1 above)
Comment (agriffard) [reply src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml]: Not if the navbar is fixed.

**Pull request merged**